DashboardClient/IDashboardDataClient.hpp

```diff
@@ -213,22 +230,34 @@ namespace Umati
                 ModelOpcUa::NodeId_t m_nodeId;
             };
 
+            class EventSubscriptionHandle {
```

Comment:
Not used?!

Reply:
Implemented this class now. Stores a clienthandle and subscriptionId for unsubscribing of Events.